pubspec.yaml

```diff
@@ -1,20 +1,19 @@
 name: flutter_redux
 description: A set of utility Widgets that Provide and Connect to a Redux Store
-version: 0.7.0
+version: 1.0.0
 author: Brian Egan <brian@brianegan.com>
 homepage: https://github.com/brianegan/flutter_redux
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  meta: ">=1.1.1 <2.0.0"
-  redux: ">=4.0.0 <5.0.0"
+  redux: ">=5.0.0 <6.0.0"
   flutter:
     sdk: flutter
 
 dev_dependencies:
-  pedantic: 1.8.0+1
+  pedantic: 1.10.0
```

Comment:
Again, rather than nail a specific version, can we allow some flexibility for the resolver?
`pedantic: ^1.10.0`

Reply:
I didn't want to change approach of author of library, but you are right flexible approach is better because in this case user of library can easy control versions of transitive dependencies without using `dependency_override` in pubspec.yaml